src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordCopyCtor.cs

```diff
@@ -115,7 +115,7 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
 
         internal static bool IsCopyConstructor(Symbol member)
         {
-            if (member is MethodSymbol { ContainingType.IsRecordStruct: false, MethodKind: MethodKind.Constructor } method)
```

Comment:
On second thought, isn't this change affecting the behavior for a copy constructor in a record struct? Looking at the [spec](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-10.0/record-structs.md#answered), there is no notion of copy constructor in record structs.